src/arch/Threads/Threads_Pthreads.cpp

```diff
@@ -208,10 +208,11 @@ MutexImpl *MakeMutex( RageMutex *pParent )
 #if defined(UNIX)
 #include <dlfcn.h>
 #include "arch/ArchHooks/ArchHooks_Unix.h"
-#elif defined(MACOSX)
-typedef int clockid_t;
-static const clockid_t CLOCK_REALTIME = 0;
-static const clockid_t CLOCK_MONOTONIC = 1;
+// commented out to allow comilation on macOS 10.12.  -dguzek
```

Comment:
To followup – I double-checked by clearing my own cmake cache, generating a fresh Xcode project file, and attempting to build again.  No dice.  These three lines that I've commented out in this PR give the following errors:

```
/Users/dan/Games/SM5/src/arch/Threads/Threads_Pthreads.cpp:212:13: Typedef redefinition with different types ('int' vs 'enum clockid_t')
/Users/dan/Games/SM5/src/arch/Threads/Threads_Pthreads.cpp:213:24: Redefinition of '_CLOCK_REALTIME' as different kind of symbol
/Users/dan/Games/SM5/src/arch/Threads/Threads_Pthreads.cpp:214:24: Redefinition of '_CLOCK_MONOTONIC' as different kind of symbol
```